server/filestore.go

```diff
@@ -7425,11 +7424,7 @@ func (fs *fileStore) State() StreamState {
 			}
 			// Add in deleted.
 			mb.dmap.Range(func(seq uint64) bool {
-				if seq < fseq {
-					mb.dmap.Delete(seq)
```

Comment:
This would fix and hide if the `mb.dmap` was bugged. But only if `fs.State()` was called.